replicatedhq/ship
# Add HelmIntro lifecycle step

What I Did
------------
- Add `HelmIntro` lifecycle step

How I Did it
------------
- Add new lifecycle step `HelmIntro` read `README.md` from `./ship`

How to verify it
------------
- Run `kustomize` subcommand with an argument of a valid chart url
- curl against the `/api/v1/lifecycle/current` endpoint
- Expect a JSON as follows with the readme populated:
```
{
  "currentStep: {
    ...
    "helmIntro": {
      "readme": ...
    },
  },
}
```

Description for the Changelog
------------
Add HelmIntro lifecycle step to send the `README.md` from the provided chart url


Picture of a Boat (not required but encouraged)
------------
![](https://cdn.bulbagarden.net/upload/8/82/SS_Anne_FRLG.png)
